composer.json

```
@@ -9,10 +9,10 @@
         }
     ],
     "require": {
-        "php": ">=8.2"
+        "php": ">=8.1"
     },
     "require-dev": {
-        "phpunit/phpunit": "~11.0",
+        "phpunit/phpunit": "^10",
```

```suggestion
        "phpunit/phpunit": "^10 || ^11",
```

I think it would make sense to support both versions, so that this library does not support an upgrade to PHPUnit 11. Wdyt?

---

That's a good idea 👍

Running the PHPUnit GitHub action with PHP 8.1 will install PHPUnit 10 and ensure that the test suite works with both versions.  
So there is no risk of using PHPUnit 11 features that are not available in PHPUnit 10.